scripts/clone-data-across-firebase-projects.ts

```diff
@@ -64,6 +54,33 @@ const destApp = initFirebaseAdminApp(DEST_PROJECT_ID, {
 // TODO: venues (owners will need to be changed)
 // TODO: check if venue already exists (be safe, don't overwrite!)
 
+const replaceSourceDomainReferences = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  obj: Record<string, any>,
```

Comment:
We are currently passing only `room` and `venue` objects in this function. Can you import the typings here? If you can it would be nice to add it and remove the `any`, if you can't then keep it as it is.

Reply:
We can definitely import the types.

I’d have to check the code here to see if it would make sense to use them in this function though (while I wrote the initial script, I didn’t write this part of it, so not as familiar with it off-hand)

Reply:
Looked into this for a little bit, while we can explicitly use `Venue | Room` as the type here, it then complicates all of the typing/etc of the rest of the code in this function, as it needs to figure out if the key is a string, then update it, etc.

We can fix this up in a future PR if we decide we care strongly about it then, but it didn't seem like it was worth the time/effort to fix here, for a fairly tightly scoped helper function in an engineering QoL script.

Reply:
> We can fix this up in a future PR....
I agree, we can create a PR in the future that cleans as many `any` usages as possible.